packages/mdx/mdx-hast-to-jsx.js

```diff
@@ -118,7 +118,8 @@ function toJSX(node, parentNode = {}, options = {}) {
                .join('')}
            </MDXTag>
   }
-}`
+}
+MDXContent.isMDXComponent = () => true`
```

Comment:
I think we can make this a property that's a boolean rather than a function:

```suggestion
MDXContent.isMDXComponent = true`
```